pandas/tests/test_categorical.py

```diff
@@ -4409,12 +4397,6 @@ def test_pickle_v0_14_1(self):
                                  name='foobar', ordered=False)
         pickle_path = os.path.join(tm.get_data_path(),
                                    'categorical_0_14_1.pickle')
-        # This code was executed once on v0.14.1 to generate the pickle:
-        #
-        # cat = Categorical(labels=np.arange(3), levels=['a', 'b', 'c', 'd'],
-        #                   name='foobar')
-        # with open(pickle_path, 'wb') as f: pickle.dump(cat, f)
-        #
```

Comment:
I think this was an attempt to test the levels in pickle. you may need to regenerate some older pickle files via ``io/tests/generate_legacy_storage_files.py`` to that we can have a regression test of these. (I am not 100% sure there was a test). but esp need it now.